otterdog/eclipse-keyple.jsonnet

```diff
@@ -30,6 +30,18 @@ orgs.newOrg('iot.keyple', 'eclipse-keyple') {
     orgs.newOrgSecret('CENTRAL_SONATYPE_TOKEN_USERNAME') {
       value: "pass:bots/iot.keyple/central.sonatype.org/token-username",
     },
+    orgs.newOrgSecret('SSH_PRIVATE_KEY') {
+      value: 'pass:bots/iot.keyple/projects-storage.eclipse.org/id_rsa',
+    },
+    orgs.newOrgSecret('SSH_PASSPHRASE') {
+      value: 'pass:bots/iot.keyple/projects-storage.eclipse.org/id_rsa.passphrase',
+    },
+    orgs.newOrgSecret('SSH_USERNAME') {
+      value: 'pass:bots/iot.keyple/projects-storage.eclipse.org/username',
+    },
+    orgs.newRepoSecret('SONAR_TOKEN') {
```

Comment:
```suggestion
    orgs.newOrgSecret('SONAR_TOKEN') {
```

Reply:
I've fixed the out of sync issue, but this is what triggers the validation issue. I'm going to to merge, this was only for awareness.